### PR TITLE
Move repo to Docker Hub

### PR DESCRIPTION
**Pull request by elliot**

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ steps:
     branches: master
     agents:
       queue: native
```
Comment (evilmarty) [inline .buildkite/pipeline.yml]: Change this to `queue: "*"`

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -38,6 +39,7 @@ steps:
...
```
Comment (evilmarty) [inline .buildkite/pipeline.yml]: And here

File: .buildkite/pipeline.yml (hunk 1 above)
Comment (elliot) [reply .buildkite/pipeline.yml]: These are build node tags

``` yaml
  - name: ":docker: Tag Staging"
    command: bin/tag staging
    branches: master
    agents:
      queue: native
      docker: builder
```

All the other steps have been switched to `queue: "*"`

Comment (evilmarty) [reply .buildkite/pipeline.yml]: I know they are the build node tags but you only need `docker: builder` and we can get rid of the `native` queue.

**Comment by elliot** [comment]
Updated for `queue: "*"`

**Pull request merged**